.gitignore

```diff
@@ -374,6 +374,8 @@ src/Hng.Web/internal-nlog.txt
 
 # AppSettings
 appsettings.*.json
+appsettings.json
+src/Hng.Web/appsettings.json
```

Comment:
Don't include app settings in git ignore.

Use it to know what env variables are required and add them as envirot variables on the server.

The format is
__ for every level down a tree